### ISSUE TITLE
Adding a --account-hw to ethers-cli & ethers-ens.

### ISSUE THREAD

**Issue by Amxx** [ethers-io/ethers-cli#5]
The documentation of ethers-cli & ethers-ens, describes numerous ways to connect to an account. however, their doesn't appear to be a option of hardware wallets (ledger in my case). I believe using `@ethersproject/hardware-wallets` this is something that could be added pretty easily. I would definitelly have usecases for it!

**Comment by ricmoo** [comment]
This will be going in soon. It was removed because the LedgerSigner was not ported to v5 yet, but now that it is, it will be added back to the CLI.

**Comment by Amxx** [comment]
Sorry to bother you, but is that, somehow, in the backlog?

**Comment by ricmoo** [comment]
Yes. I've changed this issue from discussion to enhancement to make that more clear. :)
